modules/andes-core/broker/src/main/java/org/wso2/andes/server/cluster/coordination/SlotAgent.java

```diff
@@ -36,19 +35,19 @@
 	 * @param storageQueueName name of storage queue
 	 * @param assignedNodeId Assigned node id of slot
 	 */
-	public void createSlot(long startMessageId, long endMessageId,
+	void createSlot(long startMessageId, long endMessageId,
```

Comment:
Yeah, as already mentioned, all methods in an interface are public and abstract. So, you can safely skip the redundant pubic, abstract keywords.